mtbf_driver/mtbf.py

```diff
@@ -256,6 +256,9 @@ def get_report(self):
 
     def time_up(self, signum, frame):
         self.logger.info("Signal handler called with signal" + str(signum))
+        if self.runner:
+            for t in self.runner.results:
+                t.shouldStop = True
```

Comment:
I put it in time_up() since we should stop the runner right after time_up() invoked.
unittest api provides only suite based stop function which is weird.